docs/source/checks/tabular/integrity/plot_mixed_data_types.py

```diff
@@ -2,27 +2,79 @@
 """
 Mixed Data Types
 ****************
-"""
 
-#%%
+This notebooks provides an overview for using and understanding the mixed data types check.
 
-import pandas as pd
+**Structure:**
+
+* `What are Mixed Data Types? <#what-are-mixed-data-types>`__
+* `Run the Check <#run-the-check>`__
+* `Define a Condition <#define-a-condition>`__
+
+What are Mixed Data Types?
+==========================
+Mixed data types is a situation when a column contains both string values and numeric values (either as numeric type or
+as string like "42.90"). The check detects columns which contains mixed types at any ratio, even if there is a single
+value which is different from the rest.
 
+We want to be aware of this since it may indicate a problem in the data collection pipeline, or represent a
+problem for the model's training.
```

Comment:
```suggestion
Mixed data types is when a column contains both string values and numeric values (either as numeric type or
as string like "42.90"). This may indicate a problem in the data collection pipeline, or represent a
problem situation for the model's training. 
This checks searches for columns with a mix of strings and numeric values and returns them and their respective ratios.
```